niaid.bionimbus.org/etlMapping.yaml

```diff
@@ -15,6 +15,7 @@ mappings:
           - name: ethnicity
           - name: vital_status
           - name: year_of_birth
+          - name: country_of_birth
```

Comment:
Can we rename this to `study_center` and not load the original `study_center` property?

Reply:
sure

Reply:
but, I think it better to change the dictionary, otherwise, it will be a misleading @zflamig